webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderUserDetailsFile.scala

```diff
@@ -317,7 +376,7 @@ object UserFileProcessing {
       //now, get users
       val users = ( (xml \ "user").toList.flatMap { node =>
        //for each node, check attribute name (mandatory), password  (mandatory) and role (optional)
-       (   node.attribute("name").map(_.toList.map(_.text))
+       (   node.attribute("name").map(_.toList.map(username => if (RudderConfig.rudderUsernameCaseSensitive) (username.text) else username.text.toLowerCase()))
```

Comment:
I think you should not lowercase here but in sanity log filter only sanityLoginFilter ( in fact you don't need to lowercase anymore after if you do it here, but you lost your case so and cannot display it in log messages... )

Reply:
Ok the problem is that it may be complicated to mody the user account afterward, keep it  like this